examples/k8s-health-check/README.md

```diff
@@ -74,35 +74,40 @@ livenessProbe:
 # We do not recommend adding a readiness probe under most circumstances
 ```
 
-2. Add `--http-address` and `--http-port` (optional) to your
+2. Enable the health checks by setting `--http-address` and `--http-port` (optional) to your
    proxy container configuration under `command: `.
     > [proxy_with_http_health_check.yaml](proxy_with_http_health_check.yaml#L53-L76)
 
 ```yaml
 args:
-  # If connecting from a VPC-native GKE cluster, you can use the
-  # following flag to have the proxy connect over private IP
-  # - "--private-ip"
-
-  # Enable HTTP health checks
-  - "--health-check"
-
-  # Listen on all addresses so the kubelet can reach the endpoints
-  - "--http-address=0.0.0.0"
-
-  # Set the port where the HTTP server listens
-  # - "--http-port=9090"
-
-  # Enable structured logging with LogEntry format:
-  - "--structured-logs"
-
-  # This flag specifies where the service account key can be found
-  # Remove this argument if you are using workload identity
-  - "--credentials-file=/secrets/service_account.json"
+# Replace <INSTANCE_CONNECTION_NAME> with the instance connection
+# name in the format: "project_name:region:instance_name"
+# Replace <DB_PORT> with the port that the proxy should open
+# to listen for database connections from the application
+- <INSTANCE_CONNECTION_NAME>?port=<DB_PORT>
```

Comment:
Why are we using the query string syntax instead of `--port=<DB_PORT>`?

Reply:
Or `CSQL_PROXY_PORT=<DB_PORT>` below?

Reply:
This example, like the operator, sets the port number on the instance instead of setting a
a global starting port number. I prefer to make a direct, unambiguous association between database 
instance and port number. However, if you prefer, I'll switch it to `--port=<DB_PORT>`

Reply:
I think the port flag is clear because there aren't more instances involved. It also matches our other examples.